src/current/v24.3/triggers.md

```diff
@@ -0,0 +1,505 @@
+
+~~~ sql
+CREATE TRIGGER trigger_name 
+  [ BEFORE | AFTER ] [ INSERT | UPDATE | DELETE ] ON table_name
+  FOR EACH ROW
+  [ WHEN boolean_condition ]
+  EXECUTE FUNCTION function_name(arguments)
+~~~
+
+- The trigger can activate `BEFORE` or `AFTER` any combination of `INSERT`, `UPDATE`, or `DELETE` statements is issued on a given table.
+	- `FOR EACH ROW` specifies a row-level trigger, which activates once for each row that is affected by the statements.
+	- `WHEN` specifies an optional boolean condition that determines whether the trigger activates for a given row.
+	- For details on the preceding behaviors, refer to [Trigger conditions](#trigger-conditions).
+- The [trigger function](#trigger-function), written in [PL/pgSQL]({% link {{ page.version.version }}/plpgsql.md %}), is executed each time the trigger activates. A comma-separated list of function arguments can be included.
+
+### Trigger conditions
+
+A trigger activates when one or more SQL statements is issued on a table. The statement can be [`INSERT`]({% link {{ page.version.version }}/insert.md %}), [`DELETE`]({% link {{ page.version.version }}/delete.md %}), or [`UPDATE`]({% link {{ page.version.version }}/update.md %}). 
+
+To specify more than one statement, use the `OR` clause. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TRIGGER check_value
+  BEFORE INSERT OR UPDATE ON users
+  ...
+~~~
+
+{{site.data.alerts.callout_info}}
+Triggers on `INSERT` and `UPDATE` also activate when [`UPSERT`]({% link {{ page.version.version }}/upsert.md %}) statements insert or update rows, respectively. However, `UPSERT` cannot be specified in a [`CREATE TRIGGER`]({% link {{ page.version.version }}/create-trigger.md %}) statement.
+{{site.data.alerts.end}}
+
+If `BEFORE` is specified, the trigger activates before the SQL operation. `BEFORE` triggers can be used to validate or modify data before it is inserted, or to check row values before they are updated.
+
+If `AFTER` is specified, the trigger activates after the SQL operation commits. `AFTER` triggers can be used to audit or cascade changes to other tables, thus maintaining data consistency.
+
+The `FOR EACH ROW` clause must be included after the table name. This specifies a *row-level trigger* that activates once for each table row that is affected by the SQL operations.
+
+An optional `WHEN` boolean condition can then be added. This further controls whether the trigger activates on an affected row, and is typically applied to the `OLD` or `NEW` [trigger variables](#trigger-variables). For example, the following trigger only activates if the row's `address` value was changed by the `UPDATE`:
+
```

Comment:
This might be a good place to mention that `OLD` can only be referenced in the `WHEN` clause of an `UPDATE` or `DELETE` trigger, and `NEW` only in an `INSERT` or `UPDATE` trigger.